static/templates/items/item-spell-sheet.hbs

```diff
@@ -54,14 +54,14 @@
         </div>
       </div>
 
```

Comment:
Please revert the formatting changes here

Reply:
ok,  but i added it because i think there is something wrong with closure of some nodes, so if you could double-check this file.